src/main/java/com/coherentsolutions/training/aqa/java/web/nikonova/objects/AuthenticationPage.java

```diff
@@ -0,0 +1,65 @@
+package com.coherentsolutions.training.aqa.java.web.nikonova.objects;
+
+import com.coherentsolutions.training.aqa.java.web.nikonova.browsers.BrowsersSettings;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+
+public class AuthenticationPage extends BasePage{
+
+    @FindBy(how = How.CSS, using = "#create-account_form")
```

Comment:
Please follow the same strategy for the FindBy annotation throughout the project. I mean, if you use an annotation with 1 parameter in one class, then you do not need to use the same annotation in another class with 2 parameters